---
title: "prep_metacarcoding_qm"
author: "Eily Allan"
date: "6/9/2022"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)

library(here)
library(tidyverse)
library(ggplot2)
library(gridExtra)
library(DECIPHER)
select <- dplyr::select
```

## take the salmonids from the mock community and see if the same ASVs are found in environmental samples

```{r read in mock community fasta}
salmonids <- readDNAStringSet(here("Input","mockcommunity","salmon_only_seqs.fasta"))
salmon.hashes <- names(salmonids)

rename <- dplyr::rename

```

## look at environmental asv table and hash key and find overlap with mock community fasta

```{r find asvs in both environmental samples and mock community}
all.enviro.asv.table <- read_csv(here("Input","metabarcoding", "combined.MiFish.ASV.table.csv"))
intersect(salmon.hashes, all.enviro.asv.table$Hash)

## things that are annotated
all.enviro.hash.key <- readRDS(here("Input","metabarcoding", "MiFish.all.previous.hashes.annotated.rds"))
intersect(salmon.hashes, all.enviro.hash.key$representative)

```

So 8 ASVs are overlapping. Let's see what they are and how many reads are in those 8 ASVs

```{r reads in mc asvs}

# Ok first pull all salmonid data out from the hash key 
enviro.salmon.hashes <- all.enviro.hash.key %>% 
  filter(family == "Salmonidae")
intersect(salmon.hashes, enviro.salmon.hashes$representative)

enviro.salmon.asv.table <- all.enviro.asv.table %>% 
  filter(Hash %in% enviro.salmon.hashes$representative) %>% 
  dplyr::rename(representative = Hash) %>% 
  left_join(enviro.salmon.hashes, by = "representative")

enviro.all.salmon.reads <- sum(enviro.salmon.asv.table$nReads)
# 13246010

enviro.salmon.MChashes <- enviro.salmon.asv.table %>% 
  filter(representative %in% salmon.hashes)
enviro.salmon.MChashes.reads <- sum(enviro.salmon.MChashes$nReads)
# 12684424

# WOW THAT IS GREAT- 
percent = 12684424/13246010*100
# 96%

```

So of all the salmonid reads in our environmental samples, 96% of reads are classified in one of the 8 ASVs that are in the mock community. So we don't have a lot of "Onchorynchus sp." that we can't identify. Since we know what we put into the mock community and we get 8 ASVs out of it, we should trust them.

This was just subsetting salmonids - while we are here, let's see how many reads in the environmental samples are annotated to anything (not just salmon). 

```{r check all read annotation}

enviro.all.reads <- sum(all.enviro.asv.table$nReads)
# 27352032

# annotated hashes sum of reads
enviro.annotated.reads <- all.enviro.asv.table %>% 
  filter(Hash %in% all.enviro.hash.key$representative) 
enviro.annotated.reads <- sum(enviro.annotated.reads$nReads)
# 20548265

percentannotated <- 20548265/27352032*100
# 75% not bad
```


#### write hash key and dada2 output for only salmonids with only hashes found in mock community 

```{r write file for qm model}

taxonomy.to.write <- enviro.salmon.asv.table %>% 
  filter(representative %in% salmon.hashes) %>% 
  dplyr::select(c(representative, taxon)) %>% 
  distinct()

asv.table.to.write <- all.enviro.asv.table %>% 
  filter(Hash %in% salmon.hashes) 

tax.table.to.write <- asv.table.to.write %>% 
  dplyr::rename(representative = Hash) %>% 
  left_join(taxonomy.to.write, by = "representative") %>% 
  select(-Locus) %>% 
  group_by(Sample_name, taxon) %>% 
  summarize(Nreads = sum(nReads)) %>% 
  filter(!is.na(taxon)) %>% 
  rename(species = taxon) %>% 
  filter(!str_detect(Sample_name, "Kangaroo")) %>% 
  filter(!str_detect(Sample_name, "ME")) %>% 
  filter(!str_detect(Sample_name, "NEB")) %>% 
  mutate(., Sample_name = case_when(Sample_name == "MiFish.0321.1Prt.Dn.1TR3" ~ "MiFish.0321.1Prt.Dn.1.TR3",
                             Sample_name == "MiFish.0421.3Chk.Dn.3TR2" ~ "MiFish.0421.3Chk.Dn.3.TR2",
                             Sample_name == "MiFish.0421.3Chk.Dn.3TR3" ~ "MiFish.0421.3Chk.Dn.3.TR3",
                             Sample_name == "MiFish.0821.2Brn.Dn.2TR2" ~ "MiFish.0821.2Brn.Dn.2.TR2", 
                             Sample_name == "MiFish.0821.2Brn.Dn.2TR3" ~ "MiFish.0821.2Brn.Dn.2.TR3",
                             TRUE ~ Sample_name)) %>% 
  separate(Sample_name, into=c("marker", "time", "creek", "site", "biol", "tech")) %>%
  select(-marker) %>% 
  mutate(., tech = case_when(tech == "TR2" ~ 2,
                             tech == "TR3" ~ 3,
                             is.na(tech) ~ 1)) 
  

write_rds(tax.table.to.write, file=paste0(here("Output","metabarcoding"),"/envirodata.salmonids.for.qm.RDS"))

```

Check it out by plotting

```{r plot it}


tax.table.to.write %>% 
  #filter(time == "0321") %>%  
  ggplot(aes(x = species, y = Nreads, color=site)) +
  geom_point() +  
  facet_grid(~creek ~ time, scales="free_y") +
  theme_bw() +
  scale_x_discrete(guide = guide_axis(angle = 90)) +
  ggtitle(label="MiFish - 0321")

```
